src/endpoint/devices.ts

```diff
@@ -5,24 +5,62 @@ import { Links, Status, SuccessStatusValue } from '../types'
 import { PresentationDevicePresentation } from './presentation'
 
 
-const HEADER_OVERRIDES = {Accept: 'application/vnd.smartthings+json;v=20170916'}
+const HEADER_OVERRIDES = { Accept: 'application/vnd.smartthings+json;v=20170916' }
 
 export interface CapabilityReference {
 	id: string
 	version?: number
 	status?: CapabilityStatus
 }
 
+export interface DeviceProfileReference {
+	id?: string
+}
+
+// Included for backwards compatibility (renamed to match API docs).
+// For new code, use DeviceProfileReference.
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ProfileIdentifier extends DeviceProfileReference {}
+
+export interface DeviceCategory {
+	name: string
+	categoryType: 'manufacturer' | 'user'
```

Comment:
We might want to go ahead and make this a type right away so users can reference category types explicitly.

Reply:
changed